tests/processes/ProcessingTest.py

```diff
@@ -155,6 +158,66 @@ def test_empty_run(self):
         # No global bear
         self.assertEqual(len(results[2]), 0)
 
+    def test_mixed_run(self):
+        self.sections['mixed'].append(Setting('jobs', '1'))
+        log_printer = ListLogPrinter()
+        global_bears = self.global_bears['mixed']
+        local_bears = self.local_bears['mixed']
+        bears = global_bears + local_bears
+
+        results = execute_section(self.sections['mixed'],
+                                  global_bears,
+                                  local_bears,
+                                  lambda *args: self.result_queue.put(args[2]),
+                                  None,
+                                  log_printer,
+                                  console_printer=self.console_printer)
+        self.assertIn("Bears that uses raw files can't be mixed with "
+                      'Bears that uses text files. Please move the following '
+                      'bears to their own section: ' +
+                      ', '.join(bear.name for bear in bears
+                                if not bear.USE_RAW_FILES),
+                      [log.message for log in log_printer.logs])
+
+    def test_raw_run(self):
+        self.sections['raw'].append(Setting('jobs', '1'))
+        results = execute_section(self.sections['raw'],
+                                  self.global_bears['raw'],
+                                  self.local_bears['raw'],
+                                  lambda *args: self.result_queue.put(args[2]),
+                                  None,
+                                  self.log_printer,
+                                  console_printer=self.console_printer)
+        self.assertTrue(results[0])
+
+        local_results = self.result_queue.get(timeout=0)
+        global_results = self.result_queue.get(timeout=0)
+        self.assertTrue(self.result_queue.empty())
+
+        self.assertEqual(len(local_results), 1)
```

Comment:
Should I put it into another commit? Because I'm using the same way as the other one.